packages/react-native-client/README.md

```diff
@@ -0,0 +1,402 @@
+
+# Documentation
+
+API documentation is available
+[here](https://fishjam-cloud.github.io/mobile-client-sdk/)
+
+# Installation
+
+Firstly install `react-native-client` with `yarn` or `npm`
+
+```
+yarn add @fishjam-cloud/react-native-client
+```
+
+or
+
+```
+npm install @fishjam-cloud/react-native-client
+```
+
+### Expo plugin
+
+If you're using development builds with `eas build` or bare workflow you can try
+using expo plugin to do the configuration below for you. Simply run:
+
+```
+expo install @fishjam-cloud/react-native-client
+```
+
+Add plugin to your `app.json` if it's not already added:
+
+```json
+{
+  "expo": {
+    "name": "example",
+    ...
+    "plugins": [
+      "@fishjam-cloud/react-native-client"
+    ]
+  }
+}
+```
+
+If you want to use screensharing feature, enable the following flag:
+
+```json
+{
+  "expo": {
+    "name": "example",
+    ...
+    "plugins": [
+      [
+        "@fishjam-cloud/react-native-client",
+        {
+          "setUpScreensharing": true,
+        }
+      ]
+    ]
+  }
+}
+```
+
+On bare workflow run `expo prebuild` to configure the app, then run
+`pod install`. On development build `eas build` should take care of it.
+
+### Android
+
+1. Add camera and microphone permissions to your `AndroidManifest.xml`.
+2. Rebuild the app. That's it!
+
+### iOS
+
+On iOS installation is a bit more complicated, because you need to setup a
+screen broadcast app extension for screensharing.
+
+1. Add camera and microphone permissions to your main `Info.plist`.
+   ```xml
+   <key>NSCameraUsageDescription</key>
+   <string>Allow $(PRODUCT_NAME) to use the camera</string>
+   <key>NSMicrophoneUsageDescription</key>
+   <string>Allow $(PRODUCT_NAME) to use the microphone</string>
+   ```
+2. We recommend adding `audio` background mode in `Info.plist` so that the app
+   doesn't disconnect when it's in background:
+
+```xml
+  <key>UIBackgroundModes</key>
+  <array>
+    <string>audio</string>
+  </array>
+```
+
+2. Open your `<your-project>.xcworkspace` in Xcode
+3. Create new Broadcast Upload Extension. Select File → New → Target... →
+   Broadcast Upload Extension → Next. Choose the name for the new target, select
+   Swift language and deselect "Include UI Extension".
+
+   ![New target config](./.github/images/xcode1.png)
+
+   Press Finish. In the next alert xcode will ask you if you want to activate
+   the new scheme - press Cancel.
+
+4. Configure app group. Go to "Signing & Capabilities" tab, click "+ Capability"
+   button in upper left corner and select "App Groups".
+
+   ![App groups config](./.github/images/xcode2.png)
+
+   Then in the "App Groups" add a new group or select existing. Usually group
+   name has format `group.<your-bundle-identifier>`. Verify that both app and
+   extension targets have app group and dev team set correctly.
+
+5. A new folder with app extension should appear on the left with contents like
+   this:
+
+   ![App extension files](./.github/images/xcode3.png)
```

Comment:
Are images paths ok?
On github preview it does not display it and i think it should sth like:
```suggestion
   ![App extension files](../.././.github/images/xcode3.png)
```

Reply:
Thanks! I totally overlooked it.